lua/nvim-tree/view.lua

```diff
@@ -75,6 +75,7 @@ M.View = {
     ["]c"]             = M.nvim_tree_callback("next_git_item"),
     ["-"]              = M.nvim_tree_callback("dir_up"),
     ["q"]              = M.nvim_tree_callback("close"),
+    ["?"]              = M.nvim_tree_callback("toggle_help")
```

Comment:
can we have this default to `g?`? `?` is for backwards search, I'll change this default locally, but just in case you accept that suggestion p:

Reply:
done